recipe/meta.yaml

```diff
@@ -0,0 +1,147 @@
+    - {{ cdt('libdrm-devel') }}               # [linux]
+    - {{ cdt('libglvnd-glx') }}               # [linux and aarch64]
+    - {{ cdt('libglvnd-egl') }}               # [linux and aarch64]
+    - {{ cdt('libice-devel') }}               # [linux]
+    - {{ cdt('libselinux-devel') }}           # [linux]
+    - {{ cdt('libsm-devel') }}                # [linux]
+    - {{ cdt('libx11-devel') }}               # [linux]
+    - {{ cdt('libxau-devel') }}               # [linux]
+    - {{ cdt('libxcomposite-devel') }}        # [linux]
+    - {{ cdt('libxcursor-devel') }}           # [linux]
+    - {{ cdt('libxdamage-devel') }}           # [linux]
+    - {{ cdt('libxext-devel') }}              # [linux]
+    - {{ cdt('libxfixes-devel') }}            # [linux]
+    - {{ cdt('libxi-devel') }}                # [linux]
+    - {{ cdt('libxkbfile-devel') }}           # [linux]
+    - {{ cdt('libxrandr-devel') }}            # [linux]
+    - {{ cdt('libxrender-devel') }}           # [linux]
+    - {{ cdt('libxscrnsaver-devel') }}        # [linux]
+    - {{ cdt('libxtst-devel') }}              # [linux]
+    - {{ cdt('libxxf86vm-devel') }}           # [linux]
+    - {{ cdt('mesa-libgbm-devel') }}          # [linux and aarch64]
+    - {{ cdt('mesa-libgl-devel') }}           # [linux]
+    - {{ cdt('mesa-libegl-devel') }}          # [linux]
+    - {{ cdt('mesa-dri-drivers') }}           # [linux]
+    - {{ cdt('pciutils-devel') }}             # [linux]
+    - {{ cdt('xcb-util-devel') }}             # [linux]
+    - {{ cdt('xcb-util-image-devel') }}       # [linux]
+    - {{ cdt('xcb-util-keysyms-devel') }}     # [linux]
+    - {{ cdt('xcb-util-renderutil-devel') }}  # [linux]
+    - {{ cdt('xcb-util-wm-devel') }}          # [linux]
+    - {{ cdt('xorg-x11-proto-devel') }}       # [linux]
+    - bison       # [linux]
+    - flex        # [linux]
+    - gperf       # [linux]
+    - jom         # [win]
+    - m2-bison    # [win]
+    - m2-flex     # [win]
+    - m2-gperf    # [win]
+    - m2-patch    # [win]
+    - patch       # [unix]
+    - pkg-config  # [unix]
+    - libtool     # [unix]
+    - cmake
+    - html5lib
+    - ninja
+    - nodejs
+    - perl
+
+  host:
+    - qtbase {{ version }}
+    - qtdeclarative {{ version }}
+    - qtwebchannel {{ version }}
+    # Qt tools Designer is required to compile designer plugin.
+    - qttools {{ version }}
+    - expat {{ expat }}            # [linux]
+    # Qt6 requires at least 2.10.92 for access to FCConfigGetSysRoot()
+    # https://www.freedesktop.org/software/fontconfig/fontconfig-devel/fcconfiggetsysroot.html
+    - fontconfig {{ fontconfig }}  # [linux]
+    - freetype 2                   # [linux]
+    - gtk2 2                       # [linux]
+    - icu {{ icu }}                # [linux]
+    - lcms2 2                      # [linux]
+    # Must match version of protobuf Chromium is building with or else generated messages error out on include.
+    - libprotobuf 3.20.3           # [linux]
```

Comment:
So this doesn't build with 4.25.3? That would make this package uninstallable with our recent packages depending on libprotobuf.

Reply:
It's just their internal messaging. Good catch though, this should have an `ignore_run_exports` on it to make sure those conflicts don't happen.

Reply:
>this should be in the build section or perhaps have an ignore_run_exports on it

I'm not following. libprotobuf v4.25.3 is needed at runtime if it's built against libprotobuf v4.25.3? ignoring run exports would just break the package, no?

Reply:
Unless it is not needed here to link against libprotobuf but rather to use protoc. Another way to bring in protoc without run_exports is to use the protobuf dependency rather than libprotobuf.
Any case, a qt package enforcing libprotobuf 3 would be a real issue.